examples/constraints.ipynb

```diff
@@ -1,5 +1,149 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Constrained Optimization\n",
+    "\n",
+    "Constrained optimization refers to situations in which you must for instance maximize `f`, a function of `x` and `y`, but the solution must lie in a region where for instance `x<y`.\n",
```

Comment:
these don't render properly for me (see image). I would suggest using TeX here.
![Screenshot 2022-11-25 at 15 44 05](https://user-images.githubusercontent.com/36440677/204008114-d661523e-28f1-49e5-82f1-51afef6d4d06.png)

Reply:
how odd. I will just remove the backticks I guess.
I just noticed there's quite a lot of other typos that need fixing too!